### ISSUE TITLE
[Question] Multiline input textbox

### ISSUE THREAD

**Issue by Stardox** [Microsoft/BotFramework-WebChat#1182]
Is it possible to enable the multiline support / text wrap of the textbox (_wc-shellinput_)? If so is it possible out-of-the box, for example via a setting / WebChat parameter?

After a short research I found [following issue](https://github.com/Microsoft/BotFramework-WebChat/issues/17), however it seems that it has been abandoned long time ago.

I have tried to replace the input itself with textarea element (after the initialization and rendering of the chat), however even though I achieved the multiline, it was no longer possible to send the messages anymore.

**Comment by compulim** [comment]
Stay tuned. It is coming in Web Chat v4.

**Comment by Stardox** [comment]
Thanks both of you for the responses!

In the meantime I have managed to workaround it by hiding the original `wc-shellinput` and rendering my own `TextArea` element in its place. Next up, I have hooked up my own event listener on the send button which will move the text from my own `TextArea` into the original `wc-shellinput` React component. Not the pretties approach, but will do for now. :-)

Closing my question.
